tests/controllers/excludename/matchname/chainsaw-test.yaml

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Authors of Nimbus
+
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: match-names-add-csib
+spec:
+  description: >
+    1. Adds a csib for matchNames with dev, staging
+    2. Delete staging ns
+    3. Update csib for matchNames with dev, prod
```

Comment:
The description should explain the purpose of the test and what it involves.  Currently, it seems like you've included the steps involved in performing the test instead of a general overview.

For e.g,.
https://github.com/5GSEC/nimbus/blob/5a4217460ea4adb01a0a1afab9e4634b7b924e1a/tests/controllers/securityintentbinding/create/chainsaw-test.yaml#L9-L11

Reply:
Will be adding meaningful comments and descriptions in a different PR